### PR TITLE
Feature/29689

### PR DESCRIPTION
**Pull request by kjugi**
+ Add default styles to typo 
+ Alph order in docs styles

https://dev.snowdog.pro/issues/29689

### CODE REVIEW

File: docs/docs-styles/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
```
Comment (Igloczek) [inline docs/docs-styles/_typography.scss]: Let's rename `docs-styles` to `styles`, b/c it's obvious that's only for docs.

File: docs/styles.scss [hunk 2]
```diff
@@ -48,8 +48,9 @@
 @import '../components/blocks/video-box/video-box';
```
Comment (Igloczek) [inline docs/styles.scss]: And move this file to `docs/styles` dir.

Comment (kjugi) [inline docs/styles.scss]: Moved and added in styles

File: docs/docs-styles/_typography.scss (hunk 1 above)
Comment (kjugi) [inline docs/docs-styles/_typography.scss]: Renamed

**Pull request merged**